### DISCUSSION TITLE
Blog Plugin is not generating folder structure even though it's defined in the mkdocs.yml file

### DISCUSSION THREAD

**Discussion by jaysekar** [mkdocs/mkdocs#3826] [Q&A] [answered]
plugins:
    - search
    - blog

mkdocs serve is running; I tried stopping and starting it but it's not generating the blog and posts folder structure

**Comment by Andre601** [comment]
I don't think this is a mkdocs specific question and may more be for the source of whatever blog plugin you use.

Also, **always** provide logs if something doesn't work. It helps seeing possible issues...

**Comment by jaysekar** [comment, marked as answer]
Hi, This was related to an indentation issue where the plugins was under the theme indent structure. I brought this inline to theme and it's working now.
